services/grafana-logging/6.38.1/defaults/cm.yaml

```diff
@@ -8,6 +8,12 @@ metadata:
 data:
   values.yaml: |
     ---
+    # overriding the default image due to this issue
+    # https://d2iq.atlassian.net/browse/D2IQ-94038
+    # cause by upstream bug
+    # https://github.com/grafana/grafana/issues/56087
+    image:
+      tag: 8.5.14
```

Comment:
think we need to make the same change in `project-grafana-logging` too

Reply:
done